### PR TITLE
us-28 - Message system kick event/handler

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

File: client/lib/messages/handlers/kick_event_handler.dart [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  void handleMessage(KickEvent message) {
+      int gameId = message.gameId;
+
+      String kicked = message.kicked;
+      String kickedBy = message.kickedBy;
+
+      game.players.removeWhere((p) => p.first == kicked);
+
+      if (kicked == currentUser.userName) {
+        var msg = "you have been kicked by: $kickedBy";
```
Comment (christiangoudreau) [inline client/lib/messages/handlers/kick_event_handler.dart]: Why is there a mix of 4spaces and 2 spaces?

Comment (christiangoudreau) [inline client/lib/messages/handlers/kick_event_handler.dart]: Historically, 2 spaces was for line width = 80

Isn't the line width for dart 120 as well?

Comment (Pacane) [inline client/lib/messages/handlers/kick_event_handler.dart]: "Why is there a mix of 4spaces and 2 spaces?" -- Very good question, there shouldn't be.

As for your other question, the convention for Dart is 2 spaces for indent and 80 line width.

**Comment by meriouma** [comment]
LGTM

**Pull request merged**